std_msgs/Header header
string matched
float64 probability
string[] candidates
float64[] probabilities

